src/qmflows/packages/packages.py

```diff
@@ -178,10 +183,15 @@ def get_property(self, prop: str) -> Any:
         if output_files:
             file_out = output_files[0]
             fun = getattr(import_parser(ds), ds['function'])
+
             # Read the keywords arguments from the properties dictionary
             kwargs = ds.get('kwargs', {})
             kwargs['plams_dir'] = plams_dir
-            return ignore_unused_kwargs(fun, [file_out], kwargs)
+            ret = ignore_unused_kwargs(fun, [file_out], kwargs)
+
+            # Cache the property and return
+            setattr(self, prop, ret)
```

Comment:
I like the cache idea especially for those properties that are read from plain text

Reply:
I am just a bit concerned about what would happen with the gigantic CP2K molecular orbitals array. But, I am pretty sure that the cache would work fine on most of the cases

Reply:
I think it might be possible to evaluate the size of the object and add it to the cache (or not) based on this value.

Reply:
Let's wait to see if we indeed have size problems before implementing such a feature

Reply:
Object-size checking implemented in https://github.com/SCM-NV/qmflows/pull/167/commits/e1c4e68d90eaa0074bcf448dab22b328970fb30e.